### PR TITLE
Use new datepicker

### PR DESCRIPTION
**Pull request by quisse**
## Subject

I am targeting this branch, because not backwards compatible.

## Changelog

```markdown
### Changed
- use newer datepicker
```

### CODE REVIEW

File: src/CustomerBundle/DependencyInjection/SonataCustomerExtension.php [hunk 1]
```diff
@@ -18,13 +18,24 @@
 use Symfony\Component\Config\FileLocator;
 use Symfony\Component\DependencyInjection\ContainerBuilder;
 use Symfony\Component\DependencyInjection\Loader\XmlFileLoader;
+use Symfony\Component\DependencyInjection\Extension\PrependExtensionInterface;
```
Comment (pierredup) [inline src/CustomerBundle/DependencyInjection/SonataCustomerExtension.php]: This needs to be loaded in alphabetical order

```diff
+use Symfony\Component\DependencyInjection\Extension\PrependExtensionInterface;
 use Symfony\Component\DependencyInjection\Loader\XmlFileLoader;
-use Symfony\Component\DependencyInjection\Extension\PrependExtensionInterface;
```

File: src/CustomerBundle/DependencyInjection/SonataCustomerExtension.php [hunk 2]
```diff
@@ -17,14 +17,26 @@
...
 {
+    /**
+     * {@inheritdoc}
+     */
+    public function prepend(ContainerBuilder $container): void
+    {
+        if ($container->hasExtension('twig')) {
+            // add custom form widgets
+            $container->prependExtensionConfig('twig', ['form_themes' => ['@SonataCore/Form/datepicker.html.twig']]);
+        }
```
Comment (greg0ire) [inline src/CustomerBundle/DependencyInjection/SonataCustomerExtension.php]: Why is the PR not backwards compatible and why is this necessary now, and was not before?

Comment (quisse) [reply src/CustomerBundle/DependencyInjection/SonataCustomerExtension.php]: Will look into this

Comment (quisse) [reply src/CustomerBundle/DependencyInjection/SonataCustomerExtension.php]: I guess it ís backwards compatible (I've rebased). 
Without this, the input used is just a normal text input. Am I missing something or shall i add this fragment to `Sonata\OrderBundle\DependencyInjection\SonataOrderExtension` so the bundles can be used separately?

Comment (greg0ire) [reply src/CustomerBundle/DependencyInjection/SonataCustomerExtension.php]: You can do that yeah. Since it is BC, maybe contribute it to the stable branch then?

Comment (quisse) [reply src/CustomerBundle/DependencyInjection/SonataCustomerExtension.php]: Was a bit confused but had to change the base branch here. (instead of rebasing in git 🤦‍♂)

Comment (greg0ire) [reply src/CustomerBundle/DependencyInjection/SonataCustomerExtension.php]: I think you would have to do both things :P

**Comment by greg0ire** [comment]
Thanks @quisse !
